user/caribou/module/src/AD9249EventConverter.cc

```diff
@@ -176,14 +196,16 @@ bool AD9249Event2StdEventConverter::Converting(
   // Add the plane to the StandardEvent
   d2->AddPlane(plane);
 
-  // Store frame begin and end in picoseconds
-  // FIXME USE TRIGGERID
-  d2->SetTimeBegin(0);
-  d2->SetTimeEnd(0);
-
-  // Copy event numer to trigger number:
-  // d2->SetTriggerN(ev->GetEventN());
-  // FIXME count our own trigger number - the data colelctor does weird stuff
+  // use timestamps
+  if( m_useTime ){
```

Comment:
This could be absorbed into the Corry module I guess, would make this code a bit less cluttered? No strong opinion though.

Reply:
yes, I agree. I will see to it when I start to fix the event building for the MAMI data.